### PR TITLE
nav component(WIP)

### PR DESCRIPTION
**Pull request by v-nakon**

### CODE REVIEW

File: src/components/services/siemaSlider/SiemaSlider.js [hunk 1]
```diff
@@ -16,27 +20,21 @@ class SiemaSlider extends Component {
...
         </div>
         <div className="services-navigation">
-          <div className="services-navigation-left" onClick={this.prev}>
-            <img
-              src="images/Vector_left.svg"
-              className="arrow-left"
-              alt="arrow-left"
-            />
-          </div>
+          <NavLeft onClick={this.prev} />
```
Comment (v-nakon) [inline src/components/services/siemaSlider/SiemaSlider.js]: function `prev` not work for this component. i don't understand.

File: src/components/services/siemaSlider/navLeft/NavLeft.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+import "./NavLeft.css";
+import left from "./Vector_left.svg";
+
+class NavLeft extends Component {
+  render() {
+    return (
+      <div className="services-navigation-left" onClick={this.props.prev}>
```
Comment (v-nakon) [inline src/components/services/siemaSlider/navLeft/NavLeft.js]: here i take `prev` in child component.

**Comment by v-nakon** [comment]
please, check it)

File: src/components/containerFeatures/featuresTextTitle/FeaturesTextTitle.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import "./FeaturesTextTitle.css";
+import ResultDesign from "./resultDesign/ResultDesign";
+import resultIcon from "./100 icon.svg";
+import designIcon from "./bag-icon.svg";
+
+class FeaturesTextTitle extends Component {
+  render() {
+    const resultText = "Ultimate Result";
```
Comment (byzyk) [inline src/components/containerFeatures/featuresTextTitle/FeaturesTextTitle.js]: FYI, you don't have to put these as variables here, even if you're passing them in props :)

**Pull request merged**